### PR TITLE
bes: Add version 1.7.7

### PR DESCRIPTION
**Pull request by erogy**
http://mion.faireal.net/BES/

### CODE REVIEW

File: bucket/bes.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.7.7",
+    "description": "Control per-process CPU usage",
```
Comment (Ash258) [inline bucket/bes.json:3]: image

File: bucket/bes.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "license": "GPL-2.0-only",
+    "url": "http://mion.faireal.net/archive/BES/bes_1.7.7.zip",
+    "hash": "md5:7a3c11b829b7ff2b057ca2ea7b2d6d70",
+    "extract_dir": "BES_1.7.7",
+    "pre_install": "if (!(Test-Path \"$persist_dir\\bes.ini\")) { New-Item \"$dir\\bes.ini\" | Out-Null }",
+    "bin": "bes.exe",
+    "shortcuts": [
+        [
+            "bes.exe",
+            "BES"
```
Comment (Ash258) [inline bucket/bes.json:14]: image

**Comment by erogy** [comment]
> 
> 
> What is `bes_sw.ini`?

It stores the time signature of program boot.

**Pull request merged**